app/main.py

```diff
@@ -1,6 +1,24 @@
+import requests
+import os
+
+API_KEY = os.environ.get('API_KEY')
+CITY = "Paris"
+URL = "http://api.weatherapi.com/v1/current.json"
+PAYLOAD = {"key": API_KEY, "q": CITY}
```

Comment:
why the key is q?

Reply:
"q" is API parameter, if I change to CITY , API returns an error